### PR TITLE
fix: replacing the light mode bg in to nextui-pro section

### PR DESCRIPTION
**Pull request by macci001**
Closes ENG-1636

## 📝 Description

PR changes the background image for the nextui-pro section and adds the bg for the mobile.

## Summary by CodeRabbit

- **New Features**
	- Enhanced performance with image preloading in the NextUIProSection component.
	- Introduced a new `fadeIn` animation for smoother transitions.
	- Streamlined rendering logic for the Marquee component with improved opacity effects.

- **Bug Fixes**
	- Simplified the duration prop for the Marquee component, ensuring consistent behavior.

### CODE REVIEW

**Comment by macci001** [comment]
@jrgarciadev since this section was looking bit plain on mobile, I have added the bg to mobile as well. 
We can remove it if it is not looking better.

File: apps/docs/components/marketing/nextui-pro-section.tsx [hunk 1]
```diff
@@ -168,12 +159,39 @@ export const NextUIProSection = () => {
               width={1920}
             />
           </Marquee>
+
+          <div className="absolute md:hidden inset-0 pointer-events-none top-0 z-20">
+            <div
+              className={cn(
+                "transition-opacity ease-linear duration-300 h-full w-full bg-[linear-gradient(to_left,_rgba(255,255,255,_0)_5%,_rgba(0,0,0,1)_100%)]",
```
Comment (macci001) [inline apps/docs/components/marketing/nextui-pro-section.tsx:166]: Fix the style here to remove the jarring effect.

**Review by wingkwong — CHANGES_REQUESTED** [review]
[pr4355-comment.webm](https://github.com/user-attachments/assets/2f6d3e96-e14f-4db4-8987-84725b4dcd42)

You can try to keep clicking the theme switch and compare the pro section and other sections. When switching theme, you will see pro section got a box for a sec. It should look like other sections.

File: apps/docs/components/marketing/nextui-pro-section.tsx [hunk 2]
```diff
@@ -156,25 +167,20 @@ export const NextUIProSection = () => {
...
           >
-            <img
-              alt="Hero Background"
-              className="w-full"
-              height={3255}
-              loading="lazy"
-              src={imgSrc}
-              width={1920}
-            />
+            <img alt="Hero Background" className="w-full opacity-0 animate-fadeIn" src={imgSrc} />
```
Comment (jrgarciadev) [inline apps/docs/components/marketing/nextui-pro-section.tsx:173]: @macci001 why don't we use Next.js Image component instead so we avoid preloading the images?

Comment (macci001) [reply apps/docs/components/marketing/nextui-pro-section.tsx]: Done, updated in latest commit.

**Pull request merged**